scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerController.java

```diff
@@ -614,6 +614,12 @@ public void onSuccess(String result) {
                                       json.get("FormattedMeanJobExecutionTime").isString().stringValue());
                             stats.put("TotalJobsCount", json.get("TotalJobsCount").isString().stringValue());
                             stats.put("PendingJobsCount", json.get("PendingJobsCount").isString().stringValue());
+                            stats.put("StalledJobsCount", json.get("StalledJobsCount").isString().stringValue());
+                            stats.put("PausedJobsCount", json.get("PausedJobsCount").isString().stringValue());
+                            stats.put("InErrorJobsCount", json.get("InErrorJobsCount").isString().stringValue());
+                            stats.put("KilledJobsCount", json.get("KilledJobsCount").isString().stringValue());
+                            stats.put("CancelledJobsCount", json.get("CancelledJobsCount").isString().stringValue());
+                            stats.put("FailedJobsCount", json.get("FailedJobsCount").isString().stringValue());
```

Comment:
Minor comment: to improve code clarity by limiting redundancy, you should consider factoring the code in a for structure.

Reply:
good point